src/transform.ts

```diff
@@ -12,13 +12,16 @@ export async function transformCSS(
       if (el.tagName.toLowerCase() === 'style') {
         // Handle inline stylesheets
         el.innerHTML = css;
-      } else if (el.tagName.toLowerCase() === 'link') {
+      } else if (el instanceof HTMLLinkElement) {
         // Create new link
         const blob = new Blob([css], { type: 'text/css' });
         const url = URL.createObjectURL(blob);
         const link = document.createElement('link');
         link.rel = 'stylesheet';
         link.href = url;
+        link.id = el.id;
+        link.media = el.media;
+        link.title = el.title;
         const promise = new Promise((res) => {
           link.onload = res;
```

Comment:
I'm fine including `disabled` here without a test, but we'll need to adjust the promise since `onload` won't fire for a disabled link. Something e.g.:

```suggestion
        link.disabled = el.disabled;
        const promise = new Promise((res) => {
          if (link.disabled) {
            res(true);
          } else {
            link.onload = res;
          }
```

Reply:
I was thinking about this earlier too. I wonder if it might be better to change `fetchCSS` to just ignore stylesheets with `disabled` (both `link` and `style`)

Reply:
Let's merge this, and could you open a new issue to consider what to do with `disabled` link/style elements?